script/lib/check-java-version

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+APP_FOLDER="apps"
+ROOT_DIR="$APP_FOLDER/../"
+
+parseJavaVersion() {
+  head -1 | cut -d'"' -f2 | sed 's/^1\.//' | cut -d'.' -f1
+}
+
+checkJavaVersion() {
+  runningJavaVersion=$(java -version 2>&1 | parseJavaVersion)
+  requiredJavaVersion=$(cat "$APP_FOLDER/../.java-version" | parseJavaVersion)
+
+  if [ "$runningJavaVersion" != "$requiredJavaVersion" ]; then
+    echo -e "Using wrong version of Java. Required ${requiredJavaVersion}. Running ${runningJavaVersion}."
```

Comment:
Working nicely and nice how clear it is about the problem.

Getting a double message when I run the start script, which is probably fine:
<img width="370" alt="image" src="https://github.com/guardian/typerighter/assets/34686302/14876168-6cb9-48dc-b502-efe83de42c33">

Reply:
Realised this is probably because I'm launching the rule-manager and the checker at the same time!